src/core/common/CustomLayout.js

```diff
@@ -16,25 +17,13 @@ const styles = {
         zIndex: 1,
         minHeight: '100vh',
         position: 'relative',
+        '& main': {
+            minHeight: "calc(100vh - 54px - 190px)",
```

Comment:
I advise the use of flexbox or CSS grid instead of using `calc`

Reply:
This component is flex initially. I added **minHeight** property and it solved the problem